lib/identity_cache.rb

```diff
@@ -89,8 +91,8 @@ def should_use_cache? # :nodoc:
     # == Parameters
     # +key+ A cache key string
     #
-    def fetch(key)
-      if should_use_cache?
+    def fetch(key, use_cache=should_use_cache?)
```

Comment:
This is inconsistent with fetch_multi which uses a keyword argument.  A keyword argument is probably appropriate here too